test/browser_tests/conf.js

```diff
@@ -25,10 +25,14 @@ function _chooseSuite( params, capabilities ) {
   // set the capabilities to use the essential suite.
   // This way setting the browser, for instance, won't run several
   // concurrent times.
-  if ( _paramIsSet( params.browserName ) ||
-       _paramIsSet( params.version ) ||
-       _paramIsSet( params.platform ) ) {
-    capabilities = defaultSuites.essential;
+  capabilities = defaultSuites.essential;
+  if ( !_paramIsSet( params.browserName ) &&
+       !_paramIsSet( params.version ) &&
+       !_paramIsSet( params.platform ) &&
+       ( typeof params.sauce === 'undefined' ||
+                params.sauce === 'true' ) &&
+       _isSauceCredentialSet() ) {
+    capabilities = defaultSuites.full;
```

Comment:
Yup, this looks great! Thanks for the cleanup!
How's https://github.com/cfpb/cfgov-refresh/commit/e4ef24ef8ad72c8fcca472a382c983c6695a0022 ?